templates/about/release-cycle.html

```diff
@@ -61,15 +61,17 @@ <h3>Version numbers are YY.MM</h3>
 
       <p>LTS or 'Long Term Support' releases are published every two years in April. LTS releases are the 'enterprise grade' releases of Ubuntu and are used the most. An estimated 95% of all Ubuntu installations are LTS releases.</p>
 
-      <p>Ubuntu LTS releases receive 5 years of standard security maintenance for all packages in the ‘Main’ repository. With an Ubuntu Pro subscription you get access to Expanded Security Maintenance (ESM) covering security fixes for packages in the ‘Universe’ repository (esm-apps feature). Additionally, ESM extends the lifetime of an Ubuntu LTS from 5 years to 10 years (esm-infra feature).</p>
+      <p>Ubuntu LTS releases receive 5 years of standard security maintenance for all packages in the ‘Main’ repository. With an Ubuntu Pro subscription, you get access to Expanded Security Maintenance (ESM) covering security fixes for packages in both the ‘Main’ and ‘Universe’ repositories for 10 years.
+         <a href="/support">Phone and ticket support</a> is also available and can be optionally added on top of an Ubuntu Pro subscription, covering the same packages covered by ESM, for the same time frame.
+      </p>
 
       <p>Every six months between LTS versions, Canonical publishes an interim release of Ubuntu, with {{ releases.latest.short_version }} being the latest example. These are production-quality releases and are supported for 9 months, with sufficient time provided for users to update, but these releases do not receive the long-term commitment of LTS releases.</p>
     </div>
 
     {% include "/about/release_cycles/ubuntu-eol.html" %}
 
     <div class="u-fixed-width">
-      <p>ESM requires Ubuntu Pro subscription.</p>
+      <p>ESM phone and ticket support require Ubuntu Pro subscription.</p>
```

Comment:
There's a missing "and" here:
```suggestion
      <p>ESM and phone and ticket support require Ubuntu Pro subscription.</p>
```